packages/dcos-integration-test/extra/resiliency/test_aws_cf.py

```diff
@@ -0,0 +1,75 @@
+import logging
+import os
+
+import pytest
+import requests
+import retrying
+import test_util.aws
+
+
+@pytest.fixture(scope='session')
+def provider(cluster):
+    """Interface for direct interation to provider hardware
+    Currently only supports AWS CF with AWS VPC coming soon
+    """
+    if cluster.provider != 'aws':
```

Comment:
nit: this technically should be "launch method" or something similar we pass through probably. the provider is the same for simple + advanced AWS templates.